src/core/style/layout.scss

```diff
@@ -144,6 +104,22 @@
     $flexName: 'flex-#{$name}';
   }
 
+  @if $name == sm {
+    $mutiplier: 2;
+  } @else if $name == gt-sm {
```

Comment:
Why isn't this just
```
@else {
  multiplier: 1;
}
```

What's with all the else-ifs that all do the same thing? Just to be explicit.